### PR TITLE
chapter 8

### PR DESCRIPTION
**Pull request by chan227**

### CODE REVIEW

**Comment by chan227** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by id: session[:user_id]
```
Comment (ngocvt-0484) [inline app/helpers/sessions_helper.rb]:
```suggestion
      @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
```

File: test/test_helper.rb [hunk 2]
```diff
@@ -9,5 +9,10 @@ class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
   fixtures :all
 
+  # Returns true if a test user is logged in.
+  def is_logged_in?
```
Comment (ngocvt-0484) [inline test/test_helper.rb]: dont need do test

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (chan227) [reply app/helpers/sessions_helper.rb]: done

File: test/test_helper.rb (hunk 2 above)
Comment (chan227) [reply test/test_helper.rb]: done

**Comment by chan227** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**